case-lib/lib.sh

```diff
@@ -1187,3 +1187,37 @@ perf_analyze()
     fi
 }
 
+# test-mic_privacy.sh needs to control mic privacy settings (on/off)
+# needs usbrelay package: https://github.com/darrylb123/usbrelay
+# param1: switch name
+# param2: switch state
+usbrelay_switch() {
+    switch_name=$1
+    state=$2
+    dlogi "Setting usbrelay switch $switch_name to $state."
+    if ! usbrelay "$switch_name=$state" >/dev/null 2>&1; then
+        dloge "Failed to set usbrelay switch $switch_name to $state."
+        skip_test "usbrelay is not responding or no relays detected. Check hardware connection."
+    fi
+
+    # wait for the switch to settle
+    sleep 0.5
+
+    # Display current state of the switch
+    current_state=$(usbrelay | grep "$switch_name" | awk -F= '{print $2}')
+
+    # Check if current_state is equal to the requested state
+    if [[ "$current_state" != "$state" ]]; then
```

Comment:
Nit
```suggestion
    [[ "$current_state" == "$state" ]] || {
```

Reads as "assert or die"